conda_forge_tick/migrators.xsh

```diff
@@ -0,0 +1,232 @@
+import copy
+import datetime
+import os
+import time
+import traceback
+import urllib.error
+
+import github3
+import networkx as nx
+from doctr.travis import run as doctr_run
+from pkg_resources import parse_version
+from rever.tools import (eval_version, indir, hash_url, replace_in_file)
+
+from conda_forge_tick.git_utils import (feedstock_url, feedstock_repo, fork_url,
+                                        get_repo)
+from conda_forge_tick.utils import parsed_meta_yaml
+
+
+class Migrator:
+    def filter(self, attrs):
+        """ If true don't act upon node
+
+        Parameters
+        ----------
+        attrs : dict
+            The node attributes
+
+        Returns
+        -------
+        bool :
+            True if node is to be skipped
+        """
+        # never run on archived feedstocks
+        return bool(attrs.get('archived', False))
+
+    def migrate(self, recipe_dir, attrs):
+        """Perform the migration, updating the ``meta.yaml``
+
+        Parameters
+        ----------
+        recipe_dir : str
+            The directory of the recipe
+        attrs : dict
+            The node attributes
+
+        Returns
+        -------
```

Comment:
you don't need a returns section if the function always returns None or is empty